tests/coverage/unreachable/assume_assert/main.rs

```diff
@@ -1,5 +1,11 @@
 // Copyright Kani Contributors
 // SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! This test should check that the region after `kani::assume(false)` is
+//! `UNCOVERED`. However, due to a technical limitation in `rustc`'s coverage
+//! instrumentation, only one `COVERED` region is reported for the whole
```

Comment:
Bummer :(

Reply:
Yeah, this was an unpleasant surprise for me too :(